### DISCUSSION TITLE
Legacy Token Expires

### DISCUSSION THREAD

**Discussion by damodaranr** [verdaccio/verdaccio#3172] [Q&A]
I am running verdaccio 5.2.2 in an Azure Container Instance. It is set to use Legacy token for the API. This is supposed to never expire as per the documentation. I have a mounted a storage account where the conf and storage folders are present and the packages are persisted. 
However, if the container is restarted a new token is issued and the older one no longer works. Is there a workaround for this? I would like to have a setup where need not have the developers update the token across multiple config files every once in a while. Please guide.

**Comment by juanpicado** [comment]
>   I have a mounted a storage account where the conf and storage folders are present and the packages are persisted.

The unique way the token expires is the secret withing the `.verdaccio-db.json` changes then the token generate is different. So I'd recommend check if that secret is changing.

**Reply by damodaranr** [reply]
This file .verdaccio-db.json is not created within the storage folder assigned in Azure. The packages are getting persisted in the configured cloud storage. Why is this happening? This is quite possibly the reason why the issues are occuring.
As per the documentation:
The default database file location is in the storage location. Starting with version 4.0.0, the database file name will be .verdaccio-db.json for a new installation of Verdaccio. When upgrading an existing Verdaccio server, the file name will remain .sinopia-db.json.
